apps/mobile/src/features/balances/total-balance.tsx

```diff
@@ -0,0 +1,22 @@
+import { Balance } from '@/components/balance/balance';
```

Comment:
I created this new component to contain the logic for handling fetch of `Account` + `Total` balances